index.html

```diff
@@ -1158,7 +935,7 @@ <h3>
             <a>Extension point</a>: process any proprietary and/or other
             supported members at this point in the algorithm.
           </li>
-          <li>Return <var>manifest</var>.
+          <li>Let document's <dfn>processed manifest</dfn> to be |manifest|.
```

Comment:
let and to be sounds weird together

```suggestion
          <li>Let document's <dfn>processed manifest</dfn> be a |manifest|.
```